oozie/src/test/java/org/apache/falcon/oozie/feed/OozieFeedWorkflowBuilderTest.java

```diff
@@ -99,6 +102,7 @@
     private static final String SRC_CLUSTER_PATH = "/feed/src-cluster.xml";
     private static final String TRG_CLUSTER_PATH = "/feed/trg-cluster.xml";
     private static final String FEED = "/feed/feed.xml";
+    private static final String POST_FEED = "/feed/fs-post-processing-feed.xml";
```

Comment:
reason I created a new feed because not all the test were calling builder.build function which creates the new workflow everytime. But seems like they donot need it.Will remove the feed and the extra process added for testing.